content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -588,3 +588,265 @@ As part of the Peruvian localization, besides creating credit notes from an exis
 you can also create debit Notes. For this just use the button “Add Debit Note”.
 
 By default the Debit Note is set in the document type.
+
+Guia de Remision Remitente 2.0
+------------------------------
+
+The Guía de Remisión Electrónica (Delivery Guide) is an electronic document generated by the 
+shipper to support the transportation or transfer of goods from one place to another, such 
+as a warehouse or establishment. In Odoo, there are several steps needed in configurations 
+before you can successfully use this feature.
+
+This guide is meant to be an all encompassing reference regarding this feature.
+
+Its use is mandatory and required by SUNAT for taxpayers who need to transfer their products, 
+with the exception of those under the Single Simplified Regime (Régimen Único Simplificado, or RUS).
+
+Delivery Guide Types
+********************
+
+Sender's Delivery Guide
+^^^^^^^^^^^^^^^^^^^^^^^
+
+- Issued by the *owner* of the goods (i.e. the sender) at the *beginning* of the shipment.  
+- This type of Delivery Guide is issued when a sale is made, a service is rendered (including processing), 
+goods are assigned for use or goods are transferred between premises of the same company and others. 
+- Learn more `here <https://www.gob.pe/7899-guia-de-remision>`_.
+- Included in Odoo.
+
+Carrier’s Delivery Guide
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+- It is issued by the *carrier* and must be issued by *each shipper* when the 
+*shipment is by public transport*. 
+- While the Sender’s Delivery Guide is a document that justifies the shipment of products 
+(and issued by the sender), the Carrier Delivery Guide justifies the transportation service performed by the driver (or carrier). 
+- Learn more `here <https://tefacturo.pe/blog/sunat/guia-de-remision-electronica/guia-de-remision-transportista/>`_. 
+- *Not included in Odoo*.
+
+Transportation Types
+********************
+
+Private
+^^^^^^^
+
+This option is used when the transfer of **goods is carried out by the owner themself** with 
+their own vehicles. For this case, a Sender’s Delivery Guide must be issued. 
+
+Public
+^^^^^^
+
+This option is used when the **goods are moved by an external carrier**. In this case, 
+two Delivery Guides must be issued; the Sender’s Delivery Guide and the Carrier’s Delivery Guide.
+
+Changes in Version 2.0
+**********************
+
+Direct Submission to SUNAT
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The creation of the GRE (Delivery Guide) in Odoo **must be sent directly to the SUNAT** regardless 
+of the electronic document provider: IAP, Digiflow or SUNAT.
+
+New Required Information
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Version 2.0 requires additional information in the general configuration as well as vehicles, contacts, 
+and products. In the general configurations, it will be necessary to add new credentials that 
+are acquired in the SUNAT portal.
+
+Flow
+****
+
+.. image:: peru/peru_flow_remision.png
+   :align: center 
+   :alt: Flow for Guia de Remision emission
+   
```

Comment:
@masi-odoo this section needs written content. An image should not be used in place of written instructions, can you please add instructions here?